app/controllers/wpcc_controller.rb

```diff
@@ -0,0 +1,17 @@
+class WpccController < EmbeddedToolsController
+  before_action :authenticate
+
+  protected
+
+  def authenticate
+    authenticate_or_request_with_http_basic do |username, password|
+      Authenticable.authenticate(username, password)
+    end
+  end
+
+  def category_id
+    ''
+  end
```

Comment:
After talking with Umair, the bradcrumbs for wpcc should be empty and there will be an user story to add one.